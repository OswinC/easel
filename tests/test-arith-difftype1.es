int x;
float f;

x = 3;
f = 1.3;

f = x - f;

printfl(f);
printfl(x- f -f);
