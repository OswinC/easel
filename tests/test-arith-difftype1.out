1.700000
-0.400000
